website/pages/_app.tsx

```diff
@@ -43,6 +43,7 @@ function MyApp({ Component, pageProps }: AppProps) {
         <meta
           property="og:title"
           content="LeafyGreen - MongoDB Design System"
+          key="title"
```

Comment:
since these need to be kept in sync we could keep these stored in one place
```ts
// metaTagKey.ts
export const metaTagKey = {
  Title: 'title',
} as const;

Reply:
yep sounds good

Reply:
and can also link the nextjs doc in this file ☺️

Reply:
updated